BrowserKit/Sources/Redux/Action.swift

```diff
@@ -10,14 +10,22 @@ import Foundation
 /// action, including a UUID for a particular window. If an Action is truly global in
 /// nature or can't be associated with a UUID it can send either the UUID for the active
 /// window (see `WindowManager.activeWindow`) or if needed `WindowUUID.unavailable`.
-public protocol Action {
-    var windowUUID: UUID { get }
-}
+open class Action {
+    public var windowUUID: UUID
+    public var viewUUID: UUID?
```

Comment:
What's the expectation for how `viewUUID` will be used? Are you anticipating this being a multi-purpose / context-specific value, so it would be where we store the Tab UUID for tab-related actions etc?

Reply:
The current use case I need to solve is differentiating actions for the normal tab panel and the private tab panel.
I considered making the windowUUID more generic so that it can be used at a lower scope if needed but I think there will be many cases were we have reducers that want to respond to events that aren't tied to the view the event originated from but are tied to the window or both so I think both a high level identifier and a low level one are needed. Open to suggestion for alternative approaches tho.

Reply:
I'm going to remove the viewUUID for now since I'm using the panelType on the tab use case to differentiate. We can always add it back later if another use case pops up that needs it.